js/form-validation.js

```diff
@@ -0,0 +1,15 @@
+document.getElementById("form").addEventListener('submit', (event) => {
+  let emailField = document.getElementById("email").value;
+  if (emailField.toLowerCase() === emailField) {
+    return true;
+  }
+  else {
+    var myForm = document.getElementById("form");
+    var addP = document.createElement("p");
+    addP.setAttribute('class', 'error-message');
+    addP.appendChild(document.createTextNode("Submition Faild: Pleas type your email in lowercase."));
```

Comment:
- [ ] I think it should be better to avoid a typo error, in your error-message I think you should write `please` and not `pleas` 🙃 🙃